content/guides/skills/examples.md

```diff
@@ -0,0 +1,140 @@
+---
+rank: 1
+related_endpoints: []
+related_guides:
+  - applications/custom-skills
+  - applications/custom-skills/setup
+related_pages: []
+required_guides: []
+related_resources: []
+alias_paths:
+  - /docs/explore-box-skill-samples
+---
+
+# Community Sample Skills
+
+The community sample skills lists publicly available Skills samples that have
+been generated by the Box developer community. They are split up into four main
+categories, and are designed for the processing of files within their
+associated categories:
+
+* Document Skills
+* Image Skills
+* Video Skills
+* Audio Skills
+
+<Message type='notice'>
+  Community sample skills are created and maintained by the community, and are
+  not owned or maintained by Box.
+</Message>
+
+You can see a full directory of samples on
+[Github](https://github.com/box-community).
+
+Want to submit your project? Please review the
+[Contribution Guidelines](https://github.com/box-community/community-guidelines/blob/master/.github/CONTRIBUTING.md).
+
+## Document Skills
+
+Box Skills samples for processing document files on Box.
+
+| Name | Description | Language |
+| ------ | ------ | ------ |
+| [Leverton Rental Lease Extraction][leverton] | Uses the
+[Leverton API][leverton_ml] to automatically extract fields address, landlord,
```

Comment:
Tables don't tend to like multiline, which breaks the line-length linter. I generally opt out of the linting by wrapping the table in:

```
<!-- markdownlint-disable line-length -->

...table...

<!-- markdownlint-enable line-length -->
```